ocaml/xapi/xapi_tunnel.ml

```diff
@@ -94,3 +94,32 @@ let destroy ~__context ~self =
   Xapi_pif.unplug ~__context ~self:pif ;
   Db.PIF.destroy ~__context ~self:pif ;
   Db.Tunnel.destroy ~__context ~self
+
+let determine_network_interface_for_private_network ~__context =
```

Comment:
I think this might not be the expected work for "for determine Networkd interface". I think it is related to the design here: https://info.citrite.net/pages/viewpage.action?spaceKey=xenserver&title=Design%3A+XenServer+Network+Management+Features#Design:XenServerNetworkManagementFeatures-3.2.1InterfacesofNetworkd
For the change related to that design, I would suggest first to have a quick discussion to make sure we got the same understanding and have a local agreement first.